src/content_cao.cpp

```diff
@@ -1776,6 +1776,17 @@ void GenericCAO::processMessage(const std::string &data)
 			m_nametag->nametag_color = m_prop.nametag_color;
 		}
 	}
+	else if (cmd == GENERIC_CMD_SPAWN_INFANT) {
```

Comment:
because now we have many commands, can you replace this if/elseif with switch case please ?